src/utils/constants.js

```diff
@@ -33,9 +33,18 @@ const modules = {
   SETTINGS: 'SETTINGS',
 };
 
+const api = {
+  CURRENT_VERSION: 'v1',
```

Comment:
This is the way to set the api version we want to hit, I don't think this is the best option but wasn't sure about alternatives, so any help or ideas on this would be appreciated

Reply:
This is more like a configuration right instead of constants? Would it be better if we store it inside a file that's used for configuration? e.g. `config.json`? constants is mostly used for values that we do not want to change right?

Reply:
Exactly, that's why I didn't like placing it here, where do you think this would fit better? Should I come up with a new `config.json` file?